src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -31,6 +33,8 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
             ext.outputChannel.show();
         }
     });
+    const children: EnvironmentTreeItem = <EnvironmentTreeItem>(await swaNode.loadAllChildren(context))[0]; // since it's newly created, should only have the environment tree node
+    await revealTreeItem(children.actionsTreeItem.fullId);
```

Comment:
I like the message change a lot - not sure about `revealTreeItem`, though. I think we generally try to avoid changing the focus in VS Code because that can be annoying. For example:
- If I started this from the command palette, I probably don't want to see the tree
- If I started working on my local app while the remote app was creating, I probably would get annoyed if you interrupted me to show the tree

Reply:
I think I'm leaning to getting rid of the `revealTreeItem` change.  Since I should be able to detect when the build is done, the line between the two products is clearer to draw for the user.

Before when I added this, I really wanted users to see that their build was in-progress so they could assume that this needed to finish before they could proceed.